docs/attestation.md

```diff
@@ -0,0 +1,120 @@
+      signing.enable = true;
+
+      apps = {
+        auditor.enable = true;
+        auditor.domain = "attestation.example.com";
+      };
+    }
+    ```
+    You also need to have signing enabled during build time.  Trying to build
+    for later signing with the release script doesn't build. (TODO: Probably a bug)
+    ```
+    Can't open /nix/store/a791pqlmh0npazbd6g310ga3b02mpz78-build=make-patched/target/product/security/auditor.x509.pem for reading, No such file or directory
+    140737348248832:error:02001002:system library:fopen:No such file or directory:crypto/bio/bss_file.c:69:fopen('/nix/store/a791pqlmh0npazbd6g310ga3b02mpz78-build=make-patched/target/product/security/auditor.x509.pem','r')
+    140737348248832:error:2006D080:BIO routines:BIO_new_file:no such file:crypto/bio/bss_file.c:76:
+    unable to load certificate
+    ```
+
+ 3. That's it from the Android side.  Note that the custom Auditor app will be
+    named “Robotnix Auditor”.  When you build GrapheneOS the normal Auditor app
+    will still be there, don't get confused (like I did).
+
+## Server side
+
+ 1. Before we begin we have to obtain the fingerprint of the custom Auditor app
+    and the AVB fingerprint.  To get the Auditor app fingerprint, we simply use
+    OpenSSL to extract the fingerprint of the signing certificate:
+    ```console
+    $ openssl x509 -noout -fingerprint -sha256 -in keys/auditor.x509.pem | awk -F '=' '{gsub(/:/,""); print $2}'
+    AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+    ```
+    The AVB fingerprint is a bit more tricky.  I own a Pixel 4a (sunfish) and
+    on this device the AVB fingerprint is simply the SHA256 hash of the AVB
+    key, but this is not the case on other devices.  Check the Auditor source
+    code for details.
+    ```console
+    $ sha256sum keys/sunfish/avb_pkmd.bin | awk '{print toupper($1)}'
+    BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
+    ```
+
+ 2. Now you can import Robotnix in your NixOS configuration with the
+    aforementioned fingerprints.
+    ```nix
+    { config, lib, pkgs, ... }:
+    {
+      imports = [
+        ((builtins.fetchTarball {
+          name = "robotnix";
+          url =
+            "https://github.com/danielfullmer/robotnix/archive/master.tar.gz";
+          sha256 = "0000000000000000000000000000000000000000000000000000";
+        }) + "/nixos")
+      ];
+
+      services.attestation-server = {
+        enable = true;
+        domain = "attestation.example.com";
+        deviceFamily = "sunfish";
+        signatureFingerprint = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+        avbFingerprint = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB";
+      };
+      services.nginx.virtualHosts."${config.services.attestation-server.domain}" = {
+        enableACME = true;
+        #locations."/api/create_account".return = "404"; # uncomment to disable registration
```

Comment:
I'd use the phrase "disable creating new accounts" instead of registration, because registration also refers to "device registration".